src/backend/distributed/commands/multi_copy.c

```diff
@@ -1217,6 +1228,209 @@ ReportCopyError(MultiConnection *connection, PGresult *result)
 }
 
 
+static void
+ConversionPathForTypes(Oid inputType, Oid destType, CopyCoercionData *result)
+{
+	Oid coercionFuncId = InvalidOid;
+	CoercionPathType coercionType;
```

Comment:
Maybe initialise to `COERCION_PATH_RELABELTYPE`

Reply:
I really dislike this convention of ours, we're just throwing away useful compiler errors.

Reply:
+1,000,000,000